micronaut/src/main/java/com/gluonhq/ignite/micronaut/view/View.java

```diff
@@ -0,0 +1,63 @@
+package com.gluonhq.ignite.micronaut.view;
+
+import javafx.scene.Parent;
+
+public interface View<T extends Parent> {
+
+    T getRoot();
+
+//    default WindowBuilder show() {
```

Comment:
Is there a reason for keeping the commented code?

Reply:
There was - future code. removing fro now.